test_assets/Asset_135.json

```diff
@@ -0,0 +1,40 @@
+{
+    "id": "Asset_135",
+    "name": "dna_double_stranded_decreases activity or abundance of_DDX3Y",
+    "description": "dna_double_stranded_decreases activity or abundance of_DDX3Y",
+    "tags": [],
+    "input_id": "NCBI Gene:8653",
+    "input_name": "DDX3Y",
+    "input_category": null,
+    "predicate_id": "biolink:biolink:decreases_activity_or_abundance_of",
+    "predicate_name": "biolink:decreases_activity_or_abundance_of",
```

Comment:
These have an extra biolink added

Reply:
ok I stripped the biolink - it would be good to sync up on how we want to pass the predicates - I guess you add these prefixes in the harness?